### ISSUE TITLE
`eslint-plugin-ember` doesn't work with `Glint`

### ISSUE THREAD

**Issue by boris-petrov** [ember-cli/eslint-plugin-ember#1176]
When a component extends either the Glimmer or the Classic component from [Glint](https://github.com/typed-ember/glint) (instead of the "normal" ones) - `eslint-plugin-ember` stops checking that class. It would be nice to make the two work together.

**Comment by bmish** [comment]
What rules are you referring to? Some rules do consider Glimmer components already.

**Comment by bmish** [comment]
Sorry, deleting comment, since I misread your post.

**Comment by boris-petrov** [comment]
One of the rules that are broken is `ember/require-super-in-lifecycle-hooks`:

```typescript
import Component from '@glimmer/component';

class Foo extends Component {
  public willDestroy(): void {
    console.log(1);
  }
}
```

This does give a warning. The following doesn't:

```typescript
import Component from '@glint/environment-ember-loose/glimmer-component';

class Foo extends Component {
  public willDestroy(): void {
    console.log(1);
  }
}
```

I've seen the same for the `ember/no-side-effects` rule. Perhaps all other are "broken" too.

**Comment by boris-petrov** [comment]
This issue is no longer relevant as of [Glint 0.7.4](https://github.com/typed-ember/glint/releases/tag/0.7.4) because with it "normal" imports can be used and `eslint-plugin-ember` works fine.
